tests/integration/rules/base64.phpt

```diff
@@ -20,5 +32,7 @@ try {
 }
 ?>
 --EXPECTF--
+"=c3VyZS4" must be Base64-encoded
+- "=c3VyZS4" must be Base64-encoded
```

Comment:
I'd rather have first the `check()` and then the `assert()`. It makes easier to read, IMO. How about you?